acquia_lift.module

```diff
@@ -40,27 +40,6 @@ function acquia_lift_menu() {
     'type' => MENU_CALLBACK,
     'file' => 'acquia_lift.ui.inc',
   );
-  $items['acquia_lift/personalize_in_context/start'] = array(
```

Comment:
I was just seeing those and thinking I should do a side ticket for the extra junk that was in there.  Glad you got it :-)